irondb.py

```diff
@@ -5,6 +5,7 @@
 import copy
 import flatbuffers
 
+import irondb_flatbuf
```

Comment:
Should we consider making this optional?  If someone doesn't want to use the C module they could still run the slow python native code?